examples/nlp/nmt_tutorial.py

```diff
@@ -1,37 +1,35 @@
-# Copyright (c) 2019 NVIDIA Corporation
+""" Copyright (c) 2019 NVIDIA Corporation
+See the tutorial and download the data here:
+https://nvidia.github.io/NeMo/nlp/
+neural-machine-translation.html#translation-with-pretrained-model
+"""
+import torch
+
 import nemo
-import nemo_nlp
-import logging
-from nemo.utils.lr_policies import CosineAnnealing
-from nemo_nlp.callbacks.translation import eval_iter_callback, \
-    eval_epochs_done_callback
+from nemo.utils.lr_policies import get_lr_policy
 
-console = logging.StreamHandler()
-console.setLevel(logging.INFO)
-# add the handler to the root logger
-logging.getLogger('').addHandler(console)
+import nemo_nlp
+from nemo_nlp.utils.callbacks.translation import \
+    eval_iter_callback, eval_epochs_done_callback
 
 parser = nemo.utils.NemoArgParser(description='Transformer for WMT16 En-De')
-parser.set_defaults(
-    train_dataset="train",
-    eval_datasets=["valid"],
-    checkpoint_dir="transformer_nmt",
-    optimizer="novograd",
-    # num_epochs=1000,
-    # 5 is too small - only use for test
-    num_epochs=5,
-    batch_size=4096,
-    eval_batch_size=4096,
-    lr=0.005,
-    beta1=0.95,
-    beta2=0.25,
-    weight_decay=0,
-    max_steps=50000,
-    iter_per_step=1,
-    checkpoint_save_freq=10000,
-    eval_freq=1000
-)
-parser.add_argument("--data_root", default="../../tests/data/en_de/", type=str)
+parser.set_defaults(train_dataset="train",
+                    eval_datasets=["valid"],
+                    work_dir="outputs/transformer_nmt",
+                    optimizer="novograd",
+                    num_epochs=500,  # 5 is too small - only use for test
```

Comment:
This is why CI is failing. It was changed from 5 -> 500. Can you either change CI and add a command line argument or change the default?